itests/hcatalog-unit/pom.xml

```diff
@@ -357,23 +357,4 @@
       </plugin>
     </plugins>
   </build>
-  <profiles>
-    <profile>
-      <!-- See HIVE-24473 -->
```

Comment:
Why are we removing these?

Reply:
See https://issues.apache.org/jira/browse/HIVE-24473?focusedCommentId=17254875&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-17254875

HIVE-24473 added a profile(-Dhbase.version) to make hive&hadoop3 can build with the ga hbase. But we can build with hbase without this profile now. No need it any more.